packages/devtools_app/lib/src/debugger/flutter/debugger_model.dart

```diff
@@ -6,13 +6,48 @@ import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart' hide Stack;
 import 'package:vm_service/vm_service.dart';
 
+/// A tuple of a script and an optional location.
+class ScriptLocation {
+  ScriptLocation(this.scriptRef, {this.location});
+
+  final ScriptRef scriptRef;
+
+  /// This field can be nullable.
```

Comment:
nit: This field can be nullable --> This field is nullable.

Reply:
changed